src/util.ts

```diff
@@ -375,3 +392,26 @@ export function toSnakeCase(camelCaseStr: string): string {
 export function toKebabCase(camelCaseStr: string): string {
   return camelCaseStr.replace(/[A-Z]/g, (m) => '-' + m.toLowerCase());
 }
+
+/**
+ * Retrieves the configured element base class list
+ *
+ * @param {Rule.RuleContext} context ESLint rule context
+ * @return {string[]}
+ */
+export function getElementBaseClasses(context: Rule.RuleContext): Set<string> {
+  const bases = new Set<string>([
```

Comment:
this should just be `'LitElement' really i think

Reply:
Lit also does export `ReactiveElement`, but I understand if you want to keep it to LitElement

Reply:
that's true but it doesn't necessarily have all of the features these rules test against. so probably makes sense to require a full blown lit element rather than part of it